engines/docker/engine.go

```diff
@@ -72,6 +84,15 @@ func (e *engine) PayloadSchema() schematypes.Object {
 				Description: "Command to run inside the container.",
 				Items:       schematypes.String{},
 			},
+			"devices": schematypes.Array{
+				Title:       "Devices",
+				Description: "List of host devices required.",
+				Items: schematypes.StringEnum{
+					Options: []string{
+						"video",
```

Comment:
We could just make the option of `video` be dependent on whether devices are enabled or not... anyways, nit... I'm still not sure what's ideal in either case.

Reply:
I think I would suggest that we do:
```go
devices: [
  {type: 'video'},
],
```

That way, if in some future we decide we want additional options, we can add them to the object...